src/client.ts

```diff
@@ -21,14 +21,84 @@ import {
 } from './http';
 import { generateRandomString, getTimeInSeconds } from './util';
 
-export type ClientOptions = {
+export type RequiredClientOptions = {
   clientId: string;
   clientSecret: string;
   apiHost: string;
   redirectUrl: string;
+};
+
+export type ClientOptions = RequiredClientOptions & {
   useDuoCodeAttribute?: boolean;
+  axios?: AxiosInstance;
```

Comment:
@AaronAtDuo 
I made those two new ClientOptions optional. So if someone decide to use old non-builder Client Class, they do not have to provide those two options. This should make this code change backward compatible